### ISSUE TITLE
Tuxclocker does nothing on my RX Vega

### ISSUE THREAD

**Issue by mauvray** [Lurkki14/tuxclocker#90]
On a fresh arch install, with the proper config, tuxclocker does not change anything for my GPU, whether it'd be power limit or pstates contrary to LACT. I don't see any error either in journalctl.

**Comment by Lurkki14** [comment]
What steps exactly lead to this "not working" state? Have you pressed 'Apply changes'?

**Comment by mauvray** [comment]
Yes, when I click on 'Apply changes', I get the small green highlight, but if I look  from sysfs, rocm-smi or lact, nothing changes.

**Comment by Lurkki14** [comment]
Does `cat /sys/class/drm/card*/device/hwmon/hwmon*/power1_cap` look different after changing power limit in tuxclocker?

And what about `cat /sys/class/drm/card*/device/pp_od_clk_voltage` with the pstates?

**Comment by mauvray** [comment]
Those contain values that I can set by hand without tuxclocker or with lact.

**Comment by Lurkki14** [comment]
By that I mean, can you post the contents of the files here

**Comment by mauvray** [comment]
`cat /sys/class/drm/card1/device/hwmon/hwmon1/power1_cap`
`221000000`
`cat /sys/class/drm/card1/device/pp_od_clk_voltage`
```
OD_SCLK:
0:        852Mhz        800mV
1:        991Mhz        900mV
2:       1138Mhz        950mV
3:       1269Mhz       1000mV
4:       1312Mhz       1050mV
5:       1474Mhz       1100mV
6:       1569Mhz       1100mV
7:       1622Mhz       1100mV
OD_MCLK:
0:        167Mhz        800mV
1:        500Mhz        800mV
2:        700Mhz        900mV
3:        820Mhz        950mV
OD_RANGE:
SCLK:     852MHz       2400MHz
MCLK:     167MHz       1500MHz
VDDC:     800mV        1200mV
```

**Comment by WildPenquin** [comment]
Hi,

Tuxclocker has no effect on my 6900 XT. 

This is on Arch Linux and Tuxclocker has been installed via AUR.

The UI shows as if the value had changed, but no values actually change (as shown my sensors command or looking at the nodes under /sys/class/drm/card1/device/hwmon/hwmon1. I can change power1_cap manually, but not via Tuxclocker.
